tests/RewriteVerify/Correct/Zeno.hs

```diff
@@ -720,7 +720,7 @@ p54finA hits the limit
 "p20fin" forall xs . len (sort xs) = walkNatList xs (len xs)
 "p21fin" forall n m . prop_21 n m = walkNat n True
 "p24fin" forall a b . (max a b) === a = walkNat a (b <= a)
-"p24finA" forall a b . walkNat a ((max a b) === a) = walkNat a (b <= a)
+"p24finA" forall a b . (max a b) === a = walkNat a (b <= a)
```

Comment:
I think I did this, but it should be reverted since this is identical to p24fin.

Reply:
Done